oemof_b3/model/component_attrs_update.yml

```diff
@@ -15,6 +15,25 @@ ch4-shortage:
   defaults:
     output_parameters: "{}"
 
+h2-shortage:
+  carrier: h2
+  tech: shortage
```

Comment:
Could use the tech attribute to describe the meaning of the technology (like in 'curtailment'), e.g. import

Reply:
Good idea! 
Seems like, it's not enough to just adapt `tech`. 
However, if I also adapt the name of the component e.g. `h2-import` I get an error that a facade `import.csv` is missing. `curtailment` does not have a facade in `oemoflex`, still seems to work fine. 
Could you give me a hint on that @jnnr ?

Reply:
Ah so maybe I accidentally had changed type instead - or additionally. 
Works fine now with 3d8c49a